root/etc/wallabag/parameters.template.yml

```diff
@@ -0,0 +1,63 @@
+parameters:
+    database_driver: ${SYMFONY__ENV__DATABASE_DRIVER:-pdo_sqlite}
+    database_host: ${SYMFONY__ENV__DATABASE_HOST:-127.0.0.1}
+    database_port: ${SYMFONY__ENV__DATABASE_PORT:-~}
+    database_name: ${SYMFONY__ENV__DATABASE_NAME:-symfony}
+    database_user: ${SYMFONY__ENV__DATABASE_USER:-root}
+    database_password: ${SYMFONY__ENV__DATABASE_PASSWORD:-~}
+    database_path: "%kernel.root_dir%/../data/db/wallabag.sqlite"
+    database_table_prefix: ${SYMFONY__ENV__DATABASE_TABLE_PREFIX:-wallabag_}
+    database_socket: null
+    database_charset: ${SYMFONY__ENV__DATABASE_CHARSET:-utf8}
+
+    domain_name: ${SYMFONY__ENV__DOMAIN_NAME:-https://your-wallabag-url-instance.com}
+
+    mailer_transport: ${SYMFONY__ENV__MAILER_TRANSPORT:-smtp}
+    mailer_user: ${SYMFONY__ENV__MAILER_USER:-~}
+    mailer_password: ${SYMFONY__ENV__MAILER_PASSWORD:-~}
+    mailer_host: ${SYMFONY__ENV__MAILER_HOST:-127.0.0.1}
+    mailer_port: ${SYMFONY__ENV__MAILER_PORT:-25}
+    mailer_encryption: ${SYMFONY__ENV__MAILER_ENCRYPTION:-~}
+    mailer_auth_mode: ${SYMFONY__ENV__MAILER_AUTH_MODE:-~}
+
+    locale: ${SYMFONY__ENV__LOCALE:-en}
+
+    # A secret key that's used to generate certain security-related tokens
+    secret: ${SYMFONY__ENV__SECRET:-ovmpmAWXRCabNlMgzlzFXDYmCFfzGv}
```

Comment:
We should remove the default secret here

Reply:
Can't we generate a random one each the file is _built_?

Reply:
I didn't change the behavior in this PR, Ansible install set the same secret. It's easy to generate random strings in Bash but it will be tricky to detect if it's the first run or not.
I think the best solution will be to remove the default value so this Environment Variable is mandatory. This could beak some installations so it has to be documented.

Reply:
Noted